eva_cttv_pipeline/clinvar_to_evidence_strings.py

```diff
@@ -1,280 +1,314 @@
+import itertools
 import json
 import sys
 import urllib.error
 import urllib.parse
 import urllib.request
 import codecs
+from collections import defaultdict
+from types import SimpleNamespace
 
 import jsonschema
 import xlrd
 
-from eva_cttv_pipeline import efo_term, clinvar_record, consequence_type, config
-from eva_cttv_pipeline import evidence_strings as ES
+from eva_cttv_pipeline import efo_term, clinvar_record, consequence_type, config, evidence_strings
 
 
 __author__ = 'Javier Lopez: javild@gmail.com'
 
 
+class Report:
+    def __init__(self, unavailable_efo_dict=None):
```

Comment:
Huh, I didn't remember that "gotcha". Yes, please implement it that way, or even better using a one-liner (like in https://github.com/EBIvariation/eva-cttv-pipeline/pull/14/files/2810841075c8aa8de7bb937d1b984d9e44524e23#diff-5b467890a39563d0d117a8bf104379a3R107).